code/modules/cargo/packs/emergency.dm

```diff
@@ -301,7 +301,7 @@
 
 /datum/supply_pack/emergency/weedcontrol
 	name = "Weed Control Crate"
-	desc = "Keep those invasive species OUT. Contains a scythe, gasmask, two sprays of Plant-B-Gone, and two anti-weed chemical grenades. Warranty void if used on ambrosia. Requires Hydroponics access to open."
+	desc = "Keep those invasive species' OUT. Contains a scythe, gasmask, two sprays of Plant-B-Gone, and two anti-weed chemical grenades. Warranty void if used on ambrosia. Requires Hydroponics access to open." //Skyrat change, fixed grammmatical error
```

Comment:
"Keep those invasive species OUT."
vs.
"Keep those invasive species' OUT."
You added an apostrophe, which is USUALLY for possession. There is no need for possession in this sentence.